src/org/jitsi/impl/neomedia/MediaStreamImpl.java

```diff
@@ -472,6 +472,14 @@ else if (Constants.ULPFEC.equals(encoding))
                 fecTransformEngine.setOutgoingPT(rtpPayloadType);
             }
         }
+        else if (Constants.FLEXFEC_03.equals(encoding))
+        {
+            logger.info("Creating FlexFEC transform engine");
+            FECTransformEngine flexFecTransformEngine =
+                new FECTransformEngine(FECTransformEngine.FecType.FLEXFEC_03,
+                    rtpPayloadType, rtpPayloadType, this);
+            setFecTransformEngine(flexFecTransformEngine);
+        }
```

Comment:
How does that play with ULPFEC? Do we ever want to enable both ulpfec and flexfec (I'm guessing not)?

Reply:
yeah my thought was it'd only be one or the other on a stream

Reply:
I supposed you've tested this, but this only works if setFecTransformEngine is called before the transform chain is initialized  (if fecTransformEngine is null when the chain is initialized, it will never be added to the chain).

In other places we use a wrapper to solve this problem (TransformEngineWrapper)